src/System.Reflection.Metadata/src/System/Reflection/Internal/Utilities/FileStreamReadLightUp.cs

```diff
@@ -156,7 +156,7 @@ IntPtr overlapped
             );
 
             // older Windows systems:
-            [DllImport(@"kernel32.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
+            [DllImport(@"api-ms-win-core-file-l1-1-0.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
```

Comment:
Just the presence of it will block any UWP application using this library from passing WACK and be rejected from store submission. So we need to evaluate whether or not we want to support this library on UWP and if we do then something else will need to be done.